pkg/curatedpackages/packageclient_test.go

```diff
@@ -103,3 +104,90 @@ func TestInstallPackagesFails(t *testing.T) {
 	err := tt.command.InstallPackage(tt.ctx, &tt.bundle.Spec.Packages[0], "my-harbor", "")
 	tt.Expect(err).To(MatchError(ContainSubstring("error installing package. Package exists")))
 }
+
+func TestApplyPackagesPass(t *testing.T) {
+	tt := newPackageTest(t)
+	tt.kubectl.EXPECT().ExecuteCommand(tt.ctx, gomock.Any()).Return(convertJsonToBytes(tt.bundle.Spec.Packages[0]), nil)
```

Comment:
Can we avoid using `gomock.Any()`? This is making the test a bit less useful since we are not really testing that we are running the right command (which is the only thing this method does).

Same for the other tests.

Reply:
Fixed it.